src/client/push-client.js

```diff
@@ -96,6 +116,18 @@ export default class PushClient {
     }
   }
 
+  /**
+   * This method will subscribe a use for push messaging.
+   *
+   * If permission isn't granted for push, this method will show the
+   * permissions dialog before attempting to subscribe the user to push.
+   *
+   * If an endpointUrl is supplied to the constructor, this will recieve
+   * a subscribe event.
+   *
+   * @return {Promise<PushSubscription>} Returns a Promise that
```

Comment:
I think that the word 'Returns' in the description is redundant, here and elsewhere.